openapi/components/schemas/LimitedUser.yaml

```diff
@@ -48,7 +48,6 @@ required:
   - statusDescription
   - currentAvatarImageUrl
   - currentAvatarThumbnailImageUrl
-  - fallbackAvatar
```

Comment:
This is still present within the API, why is it being removed?

Reply:
Maybe it doesn't come back on all requests anymore? They just removed it from the required fields.

Reply:
If that's the case, please provide under which conditions it is not included.

Reply:
Its not @ariesclark

Reply:
> If that's the case, please provide under which conditions it is not included. I cannot reproduce it's removal.

Idk what you're doing its not there under any endpoint for me